spec/features/idv/doc_auth/document_capture_step_spec.rb

```diff
@@ -325,8 +325,9 @@
       it 'proceeds to the next page with valid info' do
         set_up_document_capture_result(
           uuid: DocumentCaptureSession.last.uuid,
-          idv_result: { success: true, errors: {}, messages: ['message'] },
-          pii: {
+          idv_result: {
+            success: true, errors: {}, messages: ['message'],
+            pii_from_doc: {
```

Comment:
style nit, indent the contents of the hash? give each of the keys above its own line?

Reply:
👍🏼